CHANGELOG.rst

```diff
@@ -4,6 +4,22 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.1.1 (2021-06-09)
+------------------
+
+**Added**
+
+* Added a window for the image viewer
```

Comment:
when adding a feature I would expect it to be not only a patch release (then it should be 1.2.0), what do you think?

Reply:
makes sense